agriMushroom/jsonld-contexts/agri-mushroom.jsonld

```diff
@@ -0,0 +1,8 @@
+{

```

Comment:
the terms defined here are not specific to mushrooms ... it could be thought of as a (future) extension to the already existing https://github.com/smart-data-models/dataModel.Agrifood/blob/master/AgriCrop/README.md?

Reply:
or another one from https://github.com/smart-data-models/dataModel.Agrifood if appropriate

Reply:
spawn (mycélium) is specific to  mushrooms.
smartdatamodels don't have AgriCropRecord (they have a AgriParcelRecord, which is not really relevant to record information on crops).
Indeed it could be a future extension of Agrifood.